src/features/bivariate_manager/components/BivariateMatrixContainer/BivariateMatrixContainer.tsx

```diff
@@ -65,18 +69,31 @@ const BivariateMatrixContainer = ({ className }: BivariateMatrixContainerProps)
           ),
           ready: () => (
             <>
+              <PanelHeader />
               <div className={s.matrixContainer}>
                 <ConnectedBivariateMatrix ref={onRefChange} />
               </div>
               <BivariateGreetingsContainer className={s.greetings} />
             </>
           ),
         })}
-        <div className={s.topRightCorner} />
-        <div className={s.bottomRightCorner} />
       </div>
     </div>
   );
 };
 
+const PanelHeader = () => {
+  const [focusedGeometry] = useAtom(focusedGeometryAtom);
```

Comment:
I don't now for sure, will we keep a subscription and make extra updates caused by focused geomtery if bir matrix is closed?

Reply:
checked it with console.log, it won't